lib/lf/mesh/entity.h

```diff
@@ -47,6 +75,13 @@ class Entity {
   virtual base::RandomAccessRange<const Entity> SubEntities(
       char rel_codim) const = 0;
 
+  /**
+   * @brief return array of relative orientations of sub-entities
+   *        of the next hight co-dimension.
+   */
+  virtual base::RandomAccessRange<const Orientation> RelativeOrientations(
+      void) const = 0;
```

Comment:
Why not return the orientation of the entity itself? I mean `e.SubEntities()[i]->RelativeOrientation()` is not much worse than `e.RelativeOrientations()[i]` but it would be more straightforward from a conceptual point of view.

Reply:
Well, a sub-entity has not way of determining its orientation with respect to a super-entity, because all incidence information is uni-directional.